L/LAMMPS/build_tarballs.jl

```diff
@@ -58,12 +59,38 @@ products = [
     ExecutableProduct("lmp", :lmp),
 ]
 
+mpi_abis = (
+    (:mpich, PackageSpec(name="MPICH_jll"), "") ,
+    (:msmpi, PackageSpec(name="MicrosoftMPI_jll"), ""),
+    (:mpitrampoline, PackageSpec(name="MPItrampoline_jll"), "2")
+)
+
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll")),
-    Dependency(PackageSpec(name="MPItrampoline_jll"), compat="2"),
-    Dependency(PackageSpec(name="MicrosoftMPI_jll"))
+    # Dependency(PackageSpec(name="MPIPlatformTag")),
```

Comment:
@staticfloat @simonbyrne 

I am debating if this should be `MPI.jl` or a tiny single preference package.